### DISCUSSION TITLE
lima.models not founds

### DISCUSSION THREAD

**Discussion by balalatuan** [aymara/lima#159] [Q&A]
hello, I'm student from linguistique informatique. I tried to excute the code:
$ pip install --upgrade pip
$ pip install aymara==0.4.1
$ lima_models.py -l eng
$ python
but python shows that lima_models not found, could anybody help me please T.T

**Comment by hleborgne** [comment]
the script is in ./lima/lima_linguisticprocessing/src/linguisticProcessing/core/TensorFlowUnits/scripts/lima_models.py
it should have been deployed e.g in a conda environment:
$ conda create --name lima
$ conda activate lima
$ conda install pip
$ pip install --upgrade pip
$  pip install aymara==0.4.1
$ which  lima_models.py
/home/$USER/conda/DIST/envs/lima/bin/lima_models.py

**Comment by kleag** [comment]
I confirm @hleborgne answer. Personally I use venv virtual environment with `virtualenvwrapper` instead of conda.
I'm using an Ubuntu derivative, so after installing  `virtualenvwrapper` the usual way (https://virtualenvwrapper.readthedocs.io/en/latest/index.html), I do
```bash
$ mkvirtualenv lima_test
$ pip install --upgrade pip
$ pip install aymara==0.4.1
$ which lima_models.py
/home/gael/.virtualenvs/lima_test/bin/lima_models.py
$ lima_models.py -l spa
```

Could you share with us your Linux and python versions?

**Reply by balalatuan** [reply]
thank you so much for you reply!
I use anoconda on mac,i could not install aymara==0.4.1 but 0.4.0,cause pythons shows that 0.4.1 not found.
here are the errors
<img width="925" alt="WeChat0ee75a6a17c4ae1b1dee817f9db056ef" src="https://user-images.githubusercontent.com/114649620/231774743-d1a07a02-608d-47d0-8d4a-91df9dafeda3.png">
<img width="925" alt="WeChat7e24e80d566977f2b93ede5e009ba581" src="https://user-images.githubusercontent.com/114649620/231775623-2a607903-b887-46cb-8a08-1b1ca8757552.png">

**Reply by kleag** [reply]
I'm sorry, but Lima python bindings do not work on MacOS, they are Linux only.
Old versions were using a docker container to execute Lima. It should work on MacOS but it is quite complicated.
To use Lima on MacOS, you should compile from sources but as it has not been done since a long time, it will probably not work immediately.
